java/src/main/native/src/row_conversion.cu

```diff
@@ -1050,9 +1020,7 @@ copy_validity_from_rows(const size_type num_rows, const size_type num_columns,
                         size_type const *batch_row_boundaries, bitmask_type **output_nm,
                         const size_type validity_offset, device_span<const tile_info> tile_infos,
                         const int8_t *input_data) {
-  extern __shared__ int8_t shared_data[];
-  int8_t *shared_tiles[NUM_VALIDITY_TILES_PER_KERNEL_LOADED] = {
-      shared_data, shared_data + shmem_used_per_tile / 2};
+  extern __shared__ int8_t shared[];
```

Comment:
What about the same statement at line 915?

Reply:
Correct, each kernel is using shared memory and must declare it.